10-JavaScript/ajax-intro.md

```diff
@@ -1,217 +1,164 @@
+(there's more than one way to do it)
 
-![ajax](https://cms-assets.tutsplus.com/uploads%2Fusers%2F30%2Fposts%2F25099%2Fimage-1453383492163.png)
+JavaScript programs have been able to make HTTP requests since Microsoft added the functionality to Internet Explorer in 1995. It originally went by the name of XMLHttpRequest, or XHR, sometimes also called AJAX.
 
-A request is made to the server. The server processes that request and sends a response back to the client. Nothing new there. However, with AJAX, we use **JavaScript** to change the parts of a page that need to be changed, instead of reloading a whole new page.
+**AJAX** stands for **'Asynchronous JavaScript And XML'**
+  - **Asynchronous** means that you can do many things at once. With AJAX, that means you can still interact with a web page while waiting for a request/response. Your page won't freeze and with for the response to come back. Even better, you can send multiple AJAX requests at a time!
+  - **JavaScript** is the programming language that is used to make AJAX do it's thing.
+  - **XML** was originally the format responses were preferred to be sent in. However, nowadays, JSON has taken over as the preferred response format.
 
-When we make a request with AJAX, we are not asking for the entire pages's content. Instead our request is only asking for the relevant parts which we want to update.
+The original XHR interface is clunky and difficult to use, so people built wrapper libraries around them to make common tasks easier. Two examples of this are jQuery's `$.ajax` function and the standalone axios library (which we're about to learn). You can still make old-fashioned XHR requests without a library, but almost no one does.
 
+The `fetch` function was added to the core JavaScript language in the last few years as a replacement for XHR. `fetch` modern syntax and functionality without loading an external library. However, `fetch` provides a much lower-level view of HTTP requests than we need, and ends up being somewhat complex to use.
 
-## Let's Make an AJAX call!
+**In this course, we will be making AJAX requests using the axios library.**
 
-Note that in order for AJAX to work, we must make requests to a running server. It can be local or online, however we cannot make requests to a file on our computer.
+## AJAX with axios
 
+All of the JavaScript we write in this lesson and the next will reference the HTML and CSS files in the [reference/axios](reference/axios) folder. If you want to follow along, you should go ahead and copy that code now.
 
-This is how to do it using vanilla JavaScript:
+With that HTML in mind, read through the following code and see if you can figure out what's going on. Specific questions to answer:
 
-```html
-<button id="load"> Fetch! </button>
-<div id="pets"></div>
-```
+- Program Structure:
+  - When does our code make the GET request? What would we need to change if we wanted it to happen as soon as the page loads?
```

Comment:
I like this section, but maybe this is a good place to do some think-pair-sharing.  We could challenge students to answer the questions below, and find a way to trigger the errors.

Reply:
I think that is a good idea! My instinct was not to add classroom management directives into the lecture notes, but if you think it would be helpful I can.